AndroidDemo/src/main/java/com/yubico/yubikit/android/app/ui/piv/PivCertificateFragment.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2019-2022 Yubico.
+ * Copyright (C) 2019-2022,2024 Yubico.
```

Comment:
Instead of 2019-2022, 2024, should it not be 2019-2024?

Reply:
I am used to this style. Only years when the source code file was changed are listed. There was no change to this file in 2023.